Formula/w/weaviate.rb

```diff
@@ -0,0 +1,31 @@
+class Weaviate < Formula
+  desc "Open-source vector database that stores both objects and vectors"
+  homepage "https://weaviate.io/developers/weaviate/"
+  url "https://github.com/weaviate/weaviate/archive/refs/tags/v1.27.6.tar.gz"
+  sha256 "d8cad0339ccfd081be6afe1afd879f64bba224ed7ecbdb8b974853b536584330"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s
+      -w
+      -X github.com/weaviate/weaviate/usecases/build.Version=#{version}
+      -X github.com/weaviate/weaviate/usecases/build.BuildUser=#{tap.user}
+      -X github.com/weaviate/weaviate/usecases/build.BuildDate=#{time.iso8601}
+    ]
+    system "go", "build", *std_go_args(ldflags:), "./cmd/weaviate-server"
+  end
+
+  test do
+    port = free_port
+    args = ["--host", "0.0.0.0", "--port", port, "--scheme", "http"]
+    pid = fork do
+      system bin/"weaviate", *args
+    end
```

Comment:
This might be easier if you don't need to reuse them
```suggestion
    pid = fork do
      system bin/"weaviate", "--host", "0.0.0.0", "--port", port, "--scheme", "http"
    end
```

Reply:
Good point, I saw many formula using this syntax, so I reused it.
I commited&pushed your suggestion